game/scripts/npc/items/item_heart.txt

```diff
@@ -1,5 +1,5 @@
-"DOTAItems"
-{
+"DOTAItems"
+{

```

Comment:
You have the wrong line endings here:
* You have: `<CRLR>`(Windows).
* You should have `<LF>`(Unix)